aidants_connect_habilitation/templates/validation_form.html

```diff
@@ -15,32 +15,30 @@ <h2>Récapitulatif de la demande</h2>
       Il est possible de les modifier si besoin.
     </p>
     {% include "_display_org_request.html" with organisation=organisation show_all_buttons=True %}
-    <!--
+    <br>
```

Comment:
Normalement il n'y a pas besoin de `<br>` ici, la marge du h3 en-dessous devrait suffire pour être conforme à la charte, non ?

Reply:
D'après la spec HTML, les <br> sont à utiliser seulement pour les retours à la ligne qui font vraiment partie du contenu, pas pour la mise en page : 

https://www.w3.org/TR/2014/REC-html5-20141028/text-level-semantics.html#the-br-element

Reply:
En fait quand j’ai changé la position du <form> j’ai perdu l’espace qu’il y avait. J’ai donc fait cette magouille pour corriger rapidement 😬

Reply:
Ah oui c'est parce que je supprime la marge du dessus quand le h3 est le premier enfant de son parent… Une règle valable pour partout ailleurs u_u

Idées de solutions : 
- sortir le h3 du form
- faire un form qui englobe plus de choses
- laisser le form autour du bloc d'en bas, et [utiliser l'attribut form](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-form) sur le nouveau textarea